Icon Selector for Kentico Cloud (#45)
Kentico/custom-element-samples: IconSelector/README.md

# Icon Selector

In Kentico Cloud, an icon selector can be specified by using the `Multiple Choice` content element. But this lacks in usablility as the content author has to guess what icon will be displayed when a particular choice is made. The icon selector hopes to make the life of the content editor easier with an easy to use dropdown of icons.

The icon selector allows the developer to specify in the configuration what labels are displays for particular icons and what values are to saved in the content element.

> You can test it by configuring https://emmanueltissera.github.io/custom-element-samples/IconSelector/index.html url for your custom element.

Reviewer: Could you set this (and other) urls to point to the target repository? 

```suggestion
You can test it by configuring https://kentico.github.io/custom-element-samples/IconSelector/index.html url for your custom element.
```

Author: 🤦‍♂️ Fixed,

Author: Fixed

You can provide an array of icons to be displayed with their source files, labels and values.

![screenshot](https://lh3.googleusercontent.com/0zFxpQlplzKOVyNDkmRP4vtiQ1fOrlBMJzJToEYK18FpaJ9kCwuFg30CkkbDgRd6TWSlzVZeXfhzh4Fn71mDy1hjmGspP1OWkgLDzHwd9ObcYeWie7hUI2P7uHj5_wSbyYT8gXVyBA=w822-h353-no)

This outputs to:

> ```text

Reviewer: ```suggestion
```plain
```

Author: Fixed.

projectManager
```

## Configuration

The main `controlLabel` parameter is used to show the text on the dropdown.

> Your have to configure `controlLabel` and `options` parameters. There should be at least more than one option for this to work.

Reviewer: ```suggestion
You have to configure `controlLabel` and `options` parameters. There should be at least more than one option for this to work.
> Default value for the `controlLabel` is 'Please select'.
```

Author: 🤦‍♂️ Fixed,
